pkgs/build-support/remove-references-to/remove-references-to.sh

```diff
@@ -32,6 +32,6 @@ done
 
 for region in "${regions[@]}"; do
     for hook in "${fixupHooks[@]}"; do
-        eval "$hook" "$region"
+        "$hook" "$region"
     done
```

Comment:
Not sure why these were looping over this manually; `runHook fixup "$region"` should fix this with less code (and theoretically handles the case where `fixupHooks` entries are raw strings to be evaluated by the shell, which is theoretically something we can support and why it sometimes calls out to `eval`, although in practice I think that the Darwin signing thing is the only hook ever registered in this case so the whole thing is probably a little over‐engineered.)

Reply:
I have a world rebuild in the pipeline anyway, so if you want to push it untested I can fold it in to test.